bundles/org.openhab.binding.mielecloud/src/main/java/org/openhab/binding/mielecloud/internal/config/servlet/CreateBridgeServlet.java

```diff
@@ -98,8 +98,7 @@ protected String getRedirectionDestination(HttpServletRequest request) {
         try {
             accessToken = authorizationHandler.getAccessToken(bridgeUid);
         } catch (OAuthException e) {
-            logger.warn("Failed to obtain access token");
-            logger.debug("Exception details:", e);
+            logger.warn("Failed to obtain access token.", e);
```

Comment:
It seems a bit unreasonable to log the stack trace when the authentication failed. You could log the exception's message. The stack trace should only be logged in rare cases e.g. a bug in your code has been detected.

Reply:
Sorry for the inconvenience, this part of the code should have been removed. The access token is no longer part of the bridge configuration, so accessing it here is unnecessary. I removed the code.